posts/2025-03-27-the-turing-way-upgrades-to-jupyter-book-2.md

```diff
@@ -0,0 +1,34 @@
+---
+title: The Turing Way upgrades to Jupyter Book 2
+subtitle: What we liked and learned, and what we’d like to see improved
+authors: Jim, Sarah, Danny
```

Comment:
If y'all have orcids or other socials that you want to put in here, that would be awesome.

```suggestion
authors:
 - name: Jim Madge
 - name: Sarah Gibson
 - name: Danny Garside
```

Reply:
Is there documentation which lists the available socials/affiliation types?
(mastodon/fediverse?)

Reply:
Pretty out of date on that, happy for PRs:
https://mystmd.org/guide/frontmatter#frontmatter-authors

Reply:
Happy to do some work on the docs - but can you point me in the direction of the code which actually determines what the possibilities are?